docs/using.md

```diff
@@ -50,6 +51,36 @@ Starting Minishift again will restore the cluster, allowing you to continue work
 The [minishift delete](./minishift_delete.md) command is used to delete the OpenShift instance.
 This command shuts down and deletes the Minishift virtual machine. No data or state is preserved.
 
+<a name="patching-openshift-configuration"></a>
+### Patching OpenShift configuration
+
+Once you have [started](#starting-openshift) OpenShift, you can view and patch the master and
+node configuration of your OpenShift cluster.
+
+You can view the current OpenShift master configuration (_master-config.yaml_) via:
+
+```shell
+$ minishift openshift config
+```
+
+For displaying the node configuration, you can specify the `target` flag.
+For more details about the `config` command refer its [synopsis](./minishift_openshift_config.md).
+
+Let's look at[Cross-origin resource sharing](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) as an example for patching the OpenShift master configuration.
+Per default OpenShift will only allow cross origin resource requests from the IP of the
+cluster as well as localhost. This is specified via the `corsAllowedOrigins` property in the
+master configuration
+[master-config.yaml](https://docs.openshift.com/enterprise/3.0/admin_guide/master_node_configuration.html#master-configuration-files). To change this value and allow
+cross origin requests from all domains, one can execute:
+
+```
+$  minishift openshift patch --patch '{"corsAllowedOrigins": [".*"]}'
+```
+
+Per default the master configuration is targeted, but you can also path the node config
+by specifying the `target` flag. For more details about the
+`patch` command refer to its [synopsis](./minishift_openshift_patch.md).
+
 <a name="environment-variables"></a>
```

Comment:
> I tried below commands and diff does not show me much difference. Not sure if it is expected.

LOL, what do you expect? You effectively changed the default CORS from allowing only requests from localhost and 192.168.64.7:8443 to all domains (.*). And yes, you just changed a single config option :-)